openbook_posts/views/post_comments/serializer_fields.py

```diff
@@ -0,0 +1,44 @@
+from rest_framework.fields import Field
+
+
+class RepliesCountField(Field):
+    def __init__(self, **kwargs):
+        kwargs['source'] = '*'
+        kwargs['read_only'] = True
+        super(RepliesCountField, self).__init__(**kwargs)
+
+    def to_representation(self, post_comment):
+        request = self.context.get('request')
+        request_user = request.user
+
+        replies_count = request_user.get_replies_count_for_post_comment(post_comment=post_comment)
+
+        return replies_count
+
+
+class RepliesField(Field):
+    SORT_CHOICE_TO_QUERY = {
```

Comment:
This should be done outside, on the view level on a serializer. The field should get a sanitised, valid, sort choice passed through context. More on this below.